pkg/sql/crdb_internal.go

```diff
@@ -867,6 +868,46 @@ func populateSessionsTable(
 	return nil
 }
 
+// crdbInternalLocalMetricsTable exposes a snapshot of the metrics on the
+// current node.
+var crdbInternalLocalMetricsTable = virtualSchemaTable{
+	schema: `CREATE TABLE crdb_internal.node_metrics (
+  store_id 	         INT NULL,       -- the store, if any, to which this metric belongs
```

Comment:
Should we also include a `node_id` column so that it's clear when connecting through a load balancer which node the metrics are from? Many other `crdb_internal` tables that only report local node information do so ( e.g. `node_build_info`, `leases`, `node_statement_statistics`, etc).